gen/definitions/network_access_authentication_rule.yaml

```diff
@@ -85,6 +85,56 @@ attributes:
     enum_values: [contains, endsWith, equals, greaterOrEquals, greaterThan, in, ipEquals, ipGreaterThan, ipLessThan, ipNotEquals, lessOrEquals, lessThan, matches, notContains, notEndsWith, notEquals, notIn, notStartsWith, startsWith]
     description: Equality operator
     example: equals
+  - model_name: children
+    type: List
+    description: List of child conditions. `condition_type` must be one of `ConditionAndBlock`, `ConditionOrBlock`, `ConditionAttributes` or `ConditionReference`.
+    exclude_test: true
+    data_path: [rule, condition] 
+    attributes:
+      - model_name: conditionType
+        mandatory: true
+        type: String
+        enum_values: [ConditionAndBlock, ConditionAttributes, ConditionOrBlock, ConditionReference]
+        description: Indicates whether the record is the condition itself or a logical aggregation. Logical aggreation indicates that additional conditions are present under the children attribute.
+        example: ConditionAttributes
+        minimum_test_value: '"ConditionReference"'
```

Comment:
if we have excluded the parent element ("children") from testing, we can remove this line

Reply:
done